suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -233,3 +233,24 @@ tests:
       module: rbd_flatten_image_feature_disable.py
       name: Test to disable image feature when flatten operation is performed
       polarion-id: CEPH-9862
+
+  - test:
+      desc: Image migration pool namespace
+      module: readonly_namespace_image_migration.py
+      config:
+        source:
+          do_not_create_image: True
```

Comment:
where you used this? Are we really need this extra args?

Reply:
do you mean "do_not_create_image: True" ? , if yes then we are not creating the image after pool creation, we have to create a image after the namespace creation.